Source/Widgets/Geocoder/Geocoder.js

```diff
@@ -70,6 +70,11 @@ value: searchText,\
 valueUpdate: "afterkeydown",\
 disable: isSearchInProgress,\
 css: { "cesium-geocoder-input-wide" : keepExpanded || searchText.length > 0 }');
+        textBox.addEventListener('focus', function() {
+            setTimeout(function() {
```

Comment:
I just tried the latest version of Chrome/Firefox/IE and everything worked fine without the timeout, so we can remove it to start and add it back if it's ever needed.

The general strategy for Cesium development is to target the latest version of the 5 major browsers (especially since 5 of them are evergreen) and only extra code for older browsers if its really needed and has wide enough use (and we can actually replicate the broken behavior we are fixing first).